src/Disks/ObjectStorages/S3/diskSettings.cpp

```diff
@@ -63,10 +63,12 @@ std::unique_ptr<S3::Client> getClient(
     client_configuration.requestTimeoutMs = config.getUInt(config_prefix + ".request_timeout_ms", 3000);
     client_configuration.maxConnections = config.getUInt(config_prefix + ".max_connections", 100);
     client_configuration.endpointOverride = uri.endpoint;
-    client_configuration.http_keep_alive_timeout_ms
-        = config.getUInt(config_prefix + ".http_keep_alive_timeout_ms", DEFAULT_HTTP_KEEP_ALIVE_TIMEOUT * 1000);
+    client_configuration.http_keep_alive_timeout_ms = config.getUInt(
+        config_prefix + ".http_keep_alive_timeout_ms", DEFAULT_HTTP_KEEP_ALIVE_TIMEOUT * 1000);
     client_configuration.http_connection_pool_size = config.getUInt(config_prefix + ".http_connection_pool_size", 1000);
     client_configuration.wait_on_pool_size_limit = false;
+    client_configuration.s3_aggressive_timeouts = config.getUInt(
+        config_prefix + ".aggressive_timeouts", client_configuration.s3_aggressive_timeouts);
```

Comment:
You're reading from the config a single uint, why is it "timeouts" and not "timeout"?

Reply:
I have named the feature flag as `s3_aggressive_timeouts`. Here I just cut off the `s3_` prefix and it becomes `aggressive_timeouts`.
I could change the feature name in order to keep meaning after cutting `s3_` prefix.

What do you like? 
`s3_aggressive_retries` + `aggressive_retries`,
`s3_adaptive_timeouts_when_retries` + `adaptive_timeouts_when_retries`,
`s3_adaptive_timeout + `adaptive_timeout`.

Reply:
Why don't you want to use the word "use"? The name "use_aggressive_timeouts" immediately makes it clear that it's a boolean. But of course as you already have other settings it's more important to follow the existing naming rules.

Reply:
I like it
`s3_use_aggressive_timeouts` + `use_aggressive_timeouts`.
Will change in that way.

Reply:
The naming is most procrastinating part, thanks.